src/inausoft.netCLI/Commands/HelpCommandHandler.cs

```diff
@@ -45,23 +43,29 @@ public override int Run(HelpCommand command)
             }
             else
             {
-                var commandHandler = _commandHandlers.FirstOrDefault(it =>
```

Comment:
I'm thinking that more user friendly would be usage `help commandName` instead of `help command commandName`.

Reply:
I don't understand. Please elaborate.

Reply:
I understand that the current usage of help command would be as follows: (suppose that our CLI program is called `Installer`, and user want to get information about command `upgrade`)
`Installer help command upgrade`
The option name should be omitted in this situation, leaving the following call: 
`Installer help upgrade`